src/OrchardCore.Modules/OrchardCore.Admin/Startup.cs

```diff
@@ -56,4 +57,24 @@ public override void Configure(IApplicationBuilder builder, IEndpointRouteBuilde
             );
         }
     }
+
+    public class AdminPagesStartup : StartupBase
```

Comment:
Separate startup? without feature attribute?

Reply:
I will comment it